src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -16,7 +15,16 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <Compile Remove="CpuMathUtils.DotNetStandard.cs" />
+    <None Include="CpuMathUtils.DotNetStandard.cs" />
```

Comment:
Maybe add a comment and a link to [https://github.com/dotnet/project-system/issues/935](https://github.com/dotnet/project-system/issues/935) to tell people why we are doing this.

Note: I would have thought just adding an unconditional `<None Include="**/*.cs"/>` would be sufficient.  That way we only need to do this in one place, instead of each specific file.

Reply:
Agree with adding a comment - thanks.

Regarding the note: I thougth we only add a "None Include" line for those files removed from compilation, which are different for the two target frameworks.  Pulling `<None Include="**/*.cs"/>` outside of a conditional item group may require three `<None Include="**/*.cs"/>` , which are not all necessary.  But I'm not sure - please let me know.

Reply:
Any idea?  Thanks!

Reply:
This works for me:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <Configurations>Debug;Release;Debug-Intrinsics;Release-Intrinsics</Configurations>
    <UseIntrinsics Condition="'$(UseIntrinsics)' == ''">$(Configuration.EndsWith('-Intrinsics'))</UseIntrinsics>
    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
    <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
    <IncludeInPackage>Microsoft.ML.CpuMath</IncludeInPackage>
    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
    <DefineConstants>$(DefineConstants);CORECLR;PRIVATE_CONTRACTS</DefineConstants>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="..\Microsoft.ML.Core\Utilities\Contracts.cs" />

    <!-- Workaround https://github.com/dotnet/project-system/issues/935 -->
    <None Include="**/*.cs" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
    <Compile Remove="CpuMathUtils.DotNetStandard.cs" />
    <!-- This is only needed until https://github.com/dotnet/corefx/issues/31064 is addressed. -->
    <PackageReference Include="System.Runtime.Intrinsics.Experimental" Version="4.6.0-preview1-26708-04" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
    <Compile Remove="CpuMathUtils.DotNetCoreApp.cs" />
    <Compile Remove="SseIntrinsics.cs" />
  </ItemGroup>
</Project>
```